sabnzbd/utils/diskspeed.py

```diff
@@ -18,30 +18,28 @@ def diskspeedmeasure(dirname: str) -> float:
     """
     dump_data = os.urandom(_DUMP_DATA_SIZE)
     start = time.time()
-    maxtime = 0.5  # sec
+    maxtime = 3  # sec
```

Comment:
3s is a bit much as you would do two tests (incomplete + complete). 
so that 6 seconds delay with the other tests would be more than most would probably be okay with. 1s would probably be fine.